lib/AST/RequirementMachine/Diagnostics.cpp

```diff
@@ -525,12 +554,21 @@ void RewriteSystem::computeRecursiveRequirementDiagnostics(
 }
 
 void RequirementMachine::computeRequirementDiagnostics(
-    SmallVectorImpl<RequirementError> &errors, SourceLoc signatureLoc) {
+    SmallVectorImpl<RequirementError> &errors,
+    SmallVectorImpl<InverseRequirement> const& inverses,
```

Comment:
It's more idiomatic to pass an ArrayRef<InverseRequirement> instead of a const reference to a SmallVector